wordle-web/pages/game.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-row justify="center" align="center">
+    <v-col cols="12" sm="8" md="6">
+      <v-card-title class="headline justify-center">
+        <h1>Monke</h1>
+      </v-card-title>
+      <v-card class="logo py-4 d-flex justify-center">
+        <DonkeyKong />
+      </v-card>
+      <v-card>
+        <v-card-title class="headline">
+          Welcome to the game page! :)
+        </v-card-title>
+        <v-card-text>
+          <p>
+            "You saved me...why?" - Monkey
+            "mm...Monkey" - Master Oogway
+          </p>
```

Comment:
```suggestion
            "You saved me...why?" - Monkey
            "mm...Monkey" - Master Oogway
```

You don't need to wrap this in a `<p>` tag